README.md

```diff
@@ -1,3 +1,2 @@
-# devops-test-repo
-
-Repository to test github actions
+# README.md
+RAJU-81
```

Comment:
Reviewer file level comment and stated a review

Reply:
Conversion continues...